service/key/key.go

```diff
@@ -298,3 +300,17 @@ func ClusterType(installation string, obj interface{}) string {
 
 	return "workload_cluster"
 }
+
+func ApiServerAuthenticationType(ctx context.Context, k8sClient k8sclient.Interface, clusterNamespace string) (string, error) {
+	secret, err := k8sClient.K8sClient().CoreV1().Secrets(clusterNamespace).Get(ctx, APIServerCertificatesSecretName, metav1.GetOptions{})
+	if err != nil {
+		return "", err
+	}
+
+	if secret.Data["token"] != nil && len(secret.Data["token"]) > 0 {
+		return "token", nil
+	} else if (secret.Data["crt"] != nil && len(secret.Data["crt"]) > 0) && (secret.Data["key"] != nil && len(secret.Data["key"]) > 0) {
+		return "certificates", nil
+	}
+	return "vintage", nil
```

Comment:
What does "vintage" mean ?
I see no use case with this authentication type ?!

Reply:
Well, it means the vintage authentication mechanism (for KVM). It's not used but i wanted to avoid setting an empty string here

Reply:
Can you add a comment please :-)

Reply:
I made it into an error :)